web/template/index.html

```diff
@@ -25,11 +25,12 @@ <h2>Mainflux Deployment Summary</h2>
         </div>
         <div class="container">
             <div id="filter-container">
-                <form id="filter-form">
+                <form id="filter-form" onsubmit="applyFilter(event)">
```

Comment:
@SammyOina, 
Is it possible to show the selected time range above the List of Countries

Reply:
@SammyOina , If from 01-07-2023 to 01-06-2023 works but response is not as from 01-06-2023 to 01-07-2023

Reply:
ok, I'll add a check for invalid ranges